test/distributed/test_c10d.py

```diff
@@ -3825,6 +3827,51 @@ def test_broadcast_coalesced_gloo_cpu(self):
             self._test_broadcast_coalesced(process_group, device, root_rank)
 
 
+class TestDistributedNNFunctions(MultiProcessTestCase):
+    def setUp(self):
+        super(TestDistributedNNFunctions, self).setUp()
+        os.environ['WORLD_SIZE'] = str(self.world_size)
+        os.environ['MASTER_ADDR'] = '127.0.0.1'
+        os.environ['MASTER_PORT'] = "12345"
+        # This doesnt work ...
```

Comment:
do you why this does not work? I saw there are multiple other tests using this.

Reply:
The port was looked for after forking the processes, so each process was holding a different port. It is mostly WIP so I hardcoded a random port number to develop the tests.

Reply:
no need to do this anymore